docs/Development/Testing.md

```diff
@@ -8,17 +8,55 @@ The VIC Test Suite includes six main test types:
     * configuration errors - tests that address model startup and error checking.
     * restart:  tests that address model state and restart capability.
     * I/O:  tests that address model input and output functionality.
-        *  forcings come out the way the come in
+        *  forcings come out the way they come in
         *  parameter files are appropriately read in allocated.
 3.  **science**:  tests that aim to assess the model's scientific skill.  Many of these tests are compared to observations of some kind.
 4.  **examples**:  a set of examples that users may download and run.
 5.  **release**:  longer, full domain simulations performed prior to release demonstrating model output for a final release.
 
+## Test data
+
+The **system** and **examples** tests use the [VIC sample data repository](https://github.com/UW-Hydro/VIC_sample_data). This repository includes short (e.g. 10 day) test setups for the VIC image and classic drivers.
+
+The **science** and **release** tests are under development (as of April 2016).
+
+## Running the VIC Test Suite
+
+The VIC test suite uses a set of Python utilities and libraries to execute the series of tests listed above. Brief instructions for installing and running the test suite are shown below:
+
+1. Download and run the Anaconda installer: http://continuum.io/downloads
+2. Install the required dependencies:
+
+    ```Bash
+    # Create a conda virtual environment including the required dependencies
+    conda env create -n vic_test_env -f ci/requirements.yml
+
+    # Active the virtual environment
+    source activate vic_test_env
+
+    # Install the Python driver into the virtual environment
+    python ./vic/drivers/python/setup.py install
+    ```
+
+3. Run the test suite:
+
+    ```Bash
+    # Print the run_tests.py usage
+    ./tests/run_tests.py -h
+
+    # Run the test suite for the unit and examples cases for the classic driver
+    ./tests/run_tests.py unit examples \
```

Comment:
Just making sure - here `./tests/run_tests.py unit examples` is exactly the command, but users don't need to replace the "examples" by something else, right?